test/domainRegicter.js

```diff
@@ -0,0 +1,119 @@
+import { expect } from "chai";
+import pkg from 'hardhat';
+import {BigNumber} from "@ethersproject/bignumber";
+const { ethers } = pkg;
+
+describe("DomainRegister", function () {
+    let domainRegister;
+    let owner, another_address, other_address;
+    let registrationFee;
+
+    beforeEach(async function () {
```

Comment:
line 11
before much better to use beforeEach hook
you deploy your contract in every test, it is not efficient